edx_proctoring/templates/emails/proctoring_attempt_satisfactory_email.html

```diff
@@ -0,0 +1,24 @@
+{% load i18n %}
+
+<p>
+    {% blocktrans %}
+        Hi {username},
```

Comment:
Also, I'm not seeing username being passed in through the context? Is it getting into the template in a different way?